<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>File_Sorter</class>
 <widget class="QWidget" name="File_Sorter">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" colspan="2">
       <widget class="QListView" name="folderListView"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="addFolderButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT</family>
          <pointsize>12</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color:#FFF;
}

QPushButton:hover {
	color:#0096FF;
}</string>
        </property>
        <property name="text">
         <string>Add Folder</string>
        </property>
        <property name="icon">
         <iconset theme="folder-new"/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="dragDropButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT</family>
          <pointsize>12</pointsize>
          <bold>false</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: 2px dotted #888;
    border-radius: 6px;
    background-color: transparent;
    padding: 6px;
}

QPushButton:hover {
	color:#0096FF;
    border-color: #aaa;
    background-color: rgba(255, 255, 255, 0.05);
}
</string>
        </property>
        <property name="text">
         <string>Sorting Box</string>
        </property>
        <property name="icon">
         <iconset theme="go-up"/>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="folderLabel">
        <property name="font">
         <font>
          <family>Arial Rounded MT</family>
          <pointsize>18</pointsize>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Folders</string>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Icons/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
